applications/dashboard/controllers/api/UsersApiController.php

```diff
@@ -388,6 +388,10 @@ public function get_me(array $query) {
                 "description" => "Total number of unread notifications for the current user.",
                 "type" => "integer",
             ],
+            "countUnreadConversations" => [
+                "description" => "Total number of unread conversations for the current user.",
```

Comment:
As an FYI: You no longer have to put descriptions on the PHP side of endpoints because the description is in the OpenAPI YAML now.